scipy/stats/_continuous_distns.py

```diff
@@ -5800,6 +5800,14 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _fitstart(self, data, args=None):
+        if args is None:
+            args = (1.0,) * self.numargs
+        # Empirically good first estimates (see gh-10908)
+        loc = np.mean(data)
+        scale = np.mean(data)
```

Comment:
```suggestion
        # Theoretically justified estimates (see gh-10908, gh-13396)
        loc = np.min(data)
        scale = np.sqrt(np.sum((x - loc)**2) / len(x))
```
Would you check locally how this does?

Reply:
It performs a bit better. Do you have a reference that I can include as a theoretical justification?

Reply:
I can write one.

Reply:
Thanks for wrapping this up!
Probably we can find a proper section in Scipy's documentation for those MLE derivations.
@WarrenWeckesser: I saw that you were reviewing some others PRs for scipy.stats; as a core-dev, do you have suggestions for mathematical notes?

Reply:
@WarrenWeckesser has often written notes like this in a PDF outside of the SciPy repo. I think you'll find them in some of the MLE PRs. The ideal place would be the [distribution tutorials](https://docs.scipy.org/doc/scipy/reference/tutorial/stats/continuous.html#continuous-distributions-in-scipy-stats), which aren't really tutorials but lists of technical information about each distribution.

Reply:
+1 for distribution "tutorials"

Reply:
I'd add a section in the [associated Nakagami page](https://docs.scipy.org/doc/scipy/reference/tutorial/stats/continuous_nakagami.html) for the ML estimators derivation using [@mdhaber's notes](https://colab.research.google.com/drive/1MODsLLxzBRXy4KbWdS3T3QCKY5NDMWPc?usp=sharing).

Reply:
@mdhaber: I've added your notes in the companion page of Nakagami in ff6a27429c468ab587b8f0471466590ae7b42f2a.